src/components/calculator.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unused-state */
```

Comment:
Please ensure that linters are not disabled.  Linters are tools that analyze your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))). Kindly enable the disabled linters.